src/components/auth/SignInScreen.js

```diff
@@ -103,104 +103,138 @@ export default class SignInScreen extends React.Component {
-                      {i18n.t('Forgot Password?')}
+                      Entrar
                     </Text>
-                  </TouchableOpacity>
-                  {/* Loading ActivityIndicator */}
+                  }
                   {this.state.isLoading &&
                     <View>
-                      <ActivityIndicator color={COLORS.lightblue} size='large' animating={this.state.isLoading} />
+                      <ActivityIndicator color={COLORS.sandy} size='large' animating={this.state.isLoading} />
                     </View>
                   }
-                </View>
+                  </TouchableOpacity>
+                </KeyboardAvoidingView>
               </Container>
-            </View>
           </TouchableWithoutFeedback>
-        </KeyboardAvoidingView>
+        <MenuBar navigation = {this.props.navigation}/>
       </SafeAreaView>
     )
   }
 }
+
+const { width, height } = Dimensions.get('screen');
+
 const styles = StyleSheet.create({
+
+  backgroundContainer: {
+    width: width,
+    height: height,
+    backgroundColor: COLORS.sandy,
+    justifyContent: 'flex-end'
+  },
+
   container: {
     flex: 1,
-    backgroundColor: COLORS.lightgreen,
+    backgroundColor: COLORS.sandy,
     justifyContent: 'center',
-    flexDirection: 'column'
+    flexDirection: 'column',
   },
+
+  container1: {
+    flex: 2,
+    backgroundColor: 'transparent',
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+    flexDirection: 'column',
+    paddingHorizontal: 5,
+  },
+
+  container2: {
+    flex: 1,
+    backgroundColor: 'transparent',
+    justifyContent: 'flex-end',
+    flexDirection: 'column',
+  },
+
   input: {
     flex: 1,
-    fontSize: 17,
-    fontWeight: 'bold',
-    color: COLORS.lightblue,
+    fontSize: 22,
+    fontWeight: 'normal',
+    color: COLORS.forestgreen,
   },
+
   infoContainer: {
+    width: width*.75,
     position: 'absolute',
-    left: 0,
+    left: width*.25,
     right: 0,
-    flexDirection: 'row',
+    flexDirection: 'column',
     justifyContent: 'center',
     alignItems: 'center',
-    paddingHorizontal: 30,
-    backgroundColor: COLORS.lightgreen,
+    paddingHorizontal: 10,
+    backgroundColor: 'transparent',
   },
   itemStyle: {
     marginBottom: 20,
-    backgroundColor: COLORS.white,
-    borderRadius: 10,
-    borderColor: 'transparent'
-  },
-  iconStyle: {
-    color: COLORS.lightblue,
-    fontSize: 30,
-    marginRight: 15,
-    marginLeft: 15,
-    flex: 0.1
+    backgroundColor: 'transparent',
+    borderBottomColor: COLORS.forestgreen,
+    borderBottomWidth: 1,
   },
+
   buttonStyle1: {
     alignItems: 'center',
-    backgroundColor: COLORS.lightblue,
-    padding: 14,
-    marginBottom: 20,
+    justifyContent: 'center',
+    backgroundColor: COLORS.forestgreen,
+    width: width*.65,
+    height: width*.2,
     borderRadius: 10,
+    marginBottom: 30,
   },
+
   buttonText1: {
-    fontSize: 18,
+    fontSize: 22,
```

Comment:
## refer to [Scaling](https://github.com/wang1kevin1/InstitutoTerra/pull/43/files#r451776397)